Library/Formula/libantlr3c.rb

```diff
@@ -1,9 +1,12 @@
-require 'formula'
-
 class Libantlr3c < Formula
-  homepage 'http://www.antlr3.org'
-  url 'http://www.antlr3.org/download/C/libantlr3c-3.4.tar.gz'
-  sha1 'faa9ab43ab4d3774f015471c3f011cc247df6a18'
+  homepage "http://www.antlr3.org"
+  url "http://www.antlr3.org/download/C/libantlr3c-3.4.tar.gz"
+  sha1 "faa9ab43ab4d3774f015471c3f011cc247df6a18"
+  revision 1
+
+  option "with-exceptions", "Compile with support for exception handling"
+  option "with-antlrdebug", "Compile with antlr debugger in runtime"
```

Comment:
My guess is that `antlrdebug` might interact badly with adding C++-style exceptions. I think disabling it across the board would probably be fine, but I don't know how others are using this formula.

Your comment on exceptions; does kill mean I should remove the option and always compile with `-fexceptions`?